doc/nrf/introduction.rst

```diff
@@ -23,6 +23,18 @@ Every nRF Connect SDK release consists of a combination of these repositories at
 The revision of each of those repositories is determined by the current revision of the main (or manifest) repository, ``sdk-nrf``.
 It contains the SDK manifest file that enables you to manage the repositories as one code base with the west tool.
 
+About the |NCS| license
+***********************
+
+We keep licenses close to the source files.
```

Comment:
```suggestion
Licenses are close to the source files.
```
are kept? Or perhaps just 'are' is enough.

Reply:
fair point, rephrased